azure-pipelines.yml

```diff
@@ -12,13 +12,28 @@ variables:
   tests: './src/Minsk.Tests/Minsk.Tests.csproj'
 
 steps:
-- script: dotnet build --configuration $(buildConfiguration) $(sln)
+- task: UseDotNet@2
+  displayName: Use .NET 5.0 Preview
+  inputs:
+    version: 5.0.x
+    packageType: sdk
+    includePreviewVersions: true
+# We now still need .NET Core 3.1 Runtime for CI tests
+- task: UseDotNet@2
+  displayName: Install .NET Core 3.1 Runtime
+  inputs:
+    version: 3.1.x
```

Comment:
Another option is to specify your `global.json` with `rollForward` set and build based on that like so:https://github.com/fsprojects/FsLexYacc/blob/master/azure-pipelines.yml#L17-L22

Reply:
@cartermp I guess the advantage of that is that you'd be using the same settings for both local development and CI builds, without having to specify/maintaining the versions in multiple places, correct? Using this, `global.json` becomes your one source of truth (almost-ish)?

Reply:
Yep! I personally always use global.json for anything other than toy projects

Reply:
@cartermp But the rolling forward does not work very well with Preview versions unfortunenately, so there you're stuck with pinning a specific version anyways. And you still need to specify the base version-wildcard in both the Azure Pipelines YAML and the global.json if I read your link correctly?

Reply:
It _should_ work with the preview setting specified, but if that doesn't then I'd consider that a bug I suppose